lti_placements/canvas_placements.py

```diff
@@ -35,14 +29,20 @@ class CanvasLtiPlacementProcessor:
     course_count: int = 0
     placement_count: int = 0
 
-    def __init__(self, canvas_url: str, canvas_token: str):
+    def __init__(self,
+                 canvas_url: str,
+                 canvas_token: str,
+                 supported_lti_tools: Union[Dict[str, Dict[str, Union[str, int]]], None]):
         self.canvas = canvasapi.Canvas(canvas_url, canvas_token)
         self.zoom_placements = ZoomPlacements(self.canvas)
+        if not supported_lti_tools:
+            supported_lti_tools = {}
+        self.supported_lti_tools = supported_lti_tools
 
     def generate_lti_course_report(self,
                                    canvas_account_id: int,
                                    enrollment_term_ids: Union[Sequence[int], None],
-                                   add_course_ids: Union[List[int], None] = None,
+                                   add_course_ids: Union[List[int], None],
```

Comment:
Nit-pick: if you're removing the `None` default parameter, could the type hint be simplified?

Reply:
Well someone can still pass None, I just removed this default to clean up the args a little bit. I'm not sure what you mean by "type hint simplified"

Reply:
Well, you are only passing `[]` now, so I guess I just don't know why we're including the possibility of `None` anymore; no one else is using this code. This isn't a big deal; you can leave it as is.